tests/roformer/test_tokenization_roformer.py

```diff
@@ -71,3 +71,11 @@ def test_training_new_tokenizer(self):
     # can't train new_tokenizer via Tokenizers lib
     def test_training_new_tokenizer_with_special_tokens_change(self):
         pass
+
+    # can't serialise custom PreTokenizer
+    def test_save_slow_from_fast_and_reload_fast(self):
+        pass
+
+    # can't serialise custom PreTokenizer
+    def test_saving_tokenizer_trainer(self):
```

Comment:
This test fails for a different reason - as far as I can tell, saving the fast tokenizer and loading it again fails because `vocab_file` is `None` on the init:

```
path = None

    def isfile(path):
        """Test whether a path is a regular file"""
        try:
>           st = os.stat(path)
E           TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType
```

Here's an example to reproduce:

```python
from transformers import AutoTokenizer

tokenizer = AutoTokenizer.from_pretrained("junnyu/roformer_chinese_small", use_fast=True)
tokenizer.save_pretrained("./tmp/roformer", legacy_format=False)
tokenizer.from_pretrained("./tmp/roformer/")
```

Reply:
Thanks for the advice! Fixed in 1abc58a5a27a8392feccb1c39833f2e01c8ade43

With this fix, the following slow tests now all pass:

```
RUN_SLOW=1 pytest tests/roformer/test_tokenization_roformer.py -s
```